src/htable.cpp

```diff
@@ -924,24 +925,24 @@ HeadedTable::HeadedTable(QWidget *parent, int opts) : QWidget(parent)
     // synchronize horizontal scrolling of head and body
     connect(body->horizontalScrollBar(), SIGNAL(valueChanged(int)), head,
             SLOT(scrollSideways(int)));
-    fontChange(font()); // *** need for init
+    fontChange(); // *** need for init
```

Comment:
When I remove it, there's no change: the font is correctly picked up *because of the change you've made here*. Yes, before the change, it was needed.